tests/conftest.py

```diff
@@ -0,0 +1,10 @@
+import pytest
+
+import enet
+import enet.live
+import enet.test
+
+
+@pytest.fixture(params=[enet.live.LiveAPIWrapper, enet.test.TestAPIWrapper])
+def api(request):
+    return request.param(enet.WSDL_URL, enet.AUTH_TOKEN)
```

Comment:
I don't understand how this fixture works. Will it return one of the APIs depending on something? or will it parametrize the test and run it with both?